templates/alertmanagerconfig.go

```diff
@@ -133,3 +133,7 @@ var AlertmanagerConfigTemplate = promv1a1.AlertmanagerConfig{
 		},
 	},
 }
+
+func getRegexMatcher(alerts []string) string {
+	return "^" + strings.Join(alerts, "$|^") + "$"
```

Comment:
Please use the `fmt` module and string templates
```suggestion
	return fmt.Sprintf("^%s$", strings.Join(alerts, "$|^"))
```